backend/src/TodosDatabase.ts

```diff
@@ -0,0 +1,64 @@
+import { TodoDto } from './dto/todoDto';
+import { ConnectionService } from './app.connectionService';
+
+export default class TodosDatabase {
+  private static connectionService = new ConnectionService();
+
+  static async createTodo(todo: TodoDto) {
```

Comment:
add return type when it can't be directly inferred by typescript compiler (throughout)

Reply:
here i return a promise from the query. Later in the service file i type it

Reply:
i think that it returns `Promise<any>` and you can add that it returns `Promise<Todo>`